apps/dashboard/src/main/java/com/akto/action/ApiCollectionsAction.java

```diff
@@ -110,6 +134,95 @@ public String deleteMultipleCollections() {
         return SUCCESS.toUpperCase();
     }
 
+    public String fetchSensitiveInfoInCollections(){
+        this.sensitiveInfoInApiCollections = SingleTypeInfoDao.instance.getSensitiveInfoForCollections() ;
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String fetchCoverageInfoInCollections(){
+        this.testedEndpointsMaps = ApiInfoDao.instance.getCoverageCount();
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String fetchSeverityInfoInCollections(){
+        this.severityInfo = TestingRunIssuesDao.instance.getSeveritiesMapForCollections();
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String fetchLastSeenInfoInCollections(){
+        this.lastTrafficSeenMap = ApiInfoDao.instance.getLastTrafficSeen();
+        return Action.SUCCESS.toUpperCase();
+    }
+
+    public String riskScoreInfo(){
+        int oneMonthBefore = Context.now() - (60 * 60 * 24 * 30) ;
+        String computedSeverityScore = "{'$cond':[{'$gte':['$severityScore',100]},2,{'$cond':[{'$gte':['$severityScore',10]},1,{'$cond':[{'$gt':['$severityScore',0]},0.5,0]}]}]}";
+        String computedAccessTypeScore = "{ '$cond': { 'if': { '$and': [ { '$gt': [ { '$size': '$apiAccessTypes' }, 0 ] }, { '$in': ['PUBLIC', '$apiAccessTypes'] } ] }, 'then': 1, 'else': 0 } }";
```

Comment:
structure this method like the following 

public String riskScoreInfo(){ //rename fetchRiskScoreInfo

	List<Bson> pipeline = buildRiskScoreCalcPipeline()

	MongoCursor<BasicDBObject> apiCursor = buildRiskScore(pipeline)

	criticalEndpointsCount, riskScoreOfCollectionsMap = calcCriticalEndpointsCount(apiCursor)

}